packages/analytics-js/src/services/ErrorHandler/ErrorHandler.ts

```diff
@@ -29,6 +30,19 @@ class ErrorHandler implements IErrorHandler {
     this.pluginEngine = pluginEngine;
   }
 
+  attachErrorListeners() {
+    if ('addEventListener' in (globalThis as any)) {
+      (globalThis as any).addEventListener('error', (event: any) => {
```

Comment:
(globalThis as typeof Window), see how it is done in other places where globalThis is used

Reply:
Addressed